distros/kubernetes/nvsentinel/Chart.yaml

```diff
@@ -55,6 +55,9 @@ dependencies:
   - name: janitor
     version: "0.1.0"
     condition: global.janitor.enabled
+  - name: janitor-provider-kwok
```

Comment:
Let's call this janitor-provider. Ideally for the helm chart if someone want to just run on AWS without writing any additional code, they should be able to provide the correct helm values, and it should just work.